jobs/api/v4/JobsSample/CreateCompanySample.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Google.Cloud.Talent.V4Beta1;
+
+namespace GoogleCloudSamples
+{
+    internal class CreateCompanySample
+    {
+        // [START job_search_create_company_beta]
+        public static object CreateCompany(string projectId, string tenantId, string displayName, string externalId)
+        {
+            CompanyServiceClient companyServiceClient = CompanyServiceClient.Create();
+            string parent = TenantName.Format(projectId, tenantId);
+            Company company = new Company
+            {
+                DisplayName = displayName,
+                ExternalId = externalId
+            };
+
+            CreateCompanyRequest request = new CreateCompanyRequest
+            {
+                Parent = parent,
```

Comment:
nit: Is there something like `ParentAsParentName`? Then you can use:
(on line 26) `ParentName parent = TenantName.From(projectId, tenantId);`

(on line 35) `ParentAsParentName = parent;`

Reply:
I have changed all string values into Specific Objects [TenantName, ProjectName, ParentAsTenantOrProjectNameOneof ]